viewing_party/party.py

```diff
@@ -1,23 +1,105 @@
-# ------------- WAVE 1 --------------------
-
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        movie = {"title": title, "genre": genre, "rating": rating}
+    else: 
+        movie = None
+    return movie
+
+def add_to_watched(user_data, movie):
+    user_data["watched"] = [movie]
+    return user_data

```

Comment:
Ah, this is kind of a mistake on our tests but there is a bug here that they don't catch.

Line 9 here will replace the entire value of `user_data["watched"]`. That's fine if there are no entries in the list, but if there had been entries in the list, we'd lose them in favor of the new list with one entry.

Not too major and sorry for our tests missing it. That being said, usually the expected behavior of adding to a list is to append the new element.

Reply:
Plus you otherwise seem to know how to use `append` (for example in the below function) so not major enough an issue to possibly move down to Yellow or Green.